src/test/java/com/googleinterns/zoomtube/servlets/TranscriptServletTest.java

```diff
@@ -1,224 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Why is this file deleted?

Reply:
This file was originally in the branch that I'm merging to but after resolving my merge conflict, it got added over to this branch. The file still exists in transcript-store.

Reply:
Sounds good!